01/style.css

```diff
@@ -17,9 +17,9 @@
     display: block;
     content: '';
     height: 0;
-    shape-outside: polygon(0 1%, 0% 100%, 100% 0);
+    shape-outside: polygon(0 0%, 0% 100%, 100% 0);
```

Comment:
Jak wpisujemy 0 to nie musimy dawać jednostki bo 0 to zero - obojętnie czy px czy %